### PR TITLE
docs: clarify that docs are for master branch

### PR DESCRIPTION
**Pull request by thefliik**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit

## PR Type

What kind of change does this PR introduce?

```
[x] Documentation content changes
```

## What is the current behavior?

Currently, the docs linked to from the platform README are for the master branch / nightly builds. I often come back to the docs ahead of a major angular release (4->5, 5->6, 6->7) and I believe that every time I've looked at them they've reflected an unreleased api. This might be bad luck on my part, but I look at the docs infrequently enough that I always forget that they don't apply to the package version I'm using.

## What is the new behavior?

This PR adds a helpful message to the readme reminding folks that it's linking to the master branch version of the docs, and explaining how to find the docs for a specific release.

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ Become a sponsor and get your logo on our README on Github with a link to your s
 
 ## Packages
 
+***Note:** The links below will take you to the master branch version of the docs.
```
Comment (timdeschryver) [inline README.md]: I'm up for this change, but it's easier to browse via `tags`.

image

Comment (thefliik) [reply README.md]: @timdeschryver right you are! Didn't know about that option. I updated the suggestion text.

**Comment by thefliik** [comment]
Feedback addressed

**Comment by brandonroberts** [comment]
We will be moving the docs to an official site soon, so these caveats will be obsolete. The docs will be based off the current published release instead of master.

**Comment by thefliik** [comment]
@brandonroberts Awesome! Ok, feel free to close. Though if "soon" might turn out to be longer then anticipated, I'd advocate for merging this change now and then simply removing it when the time comes. Might help someone without an apparent downside.

**Comment by timdeschryver** [comment]
I was thinking about putting this message until the site is published, because we're getting some issues/questions because users are using unreleased features.

**Pull request merged**